client/src/index.jsx

```diff
@@ -1,7 +1,55 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import RelatedContent from './components/container.jsx';
+import axios from 'axios';
+import { MainContainer, Title, Browse } from './components/styles/auxstyles.js';
+const url = '/api/dlc/';
 
 
 
-ReactDOM.render(<RelatedContent />, document.getElementById('app'));
\ No newline at end of file
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      gameId: 2,
+      data: [{}]
```

Comment:
For the moment, defaulting to 2 is probably fine, but for the real version you'll probably want to figure out what to do if this information is legitimately missing.